llvm/lib/Transforms/Vectorize/VPlanConstruction.cpp

```diff
@@ -662,7 +673,7 @@ void VPlanTransforms::attachCheckBlock(VPlan &Plan, Value *Cond,
       VPBuilder(CheckBlockVPBB)
           .createNaryOp(
               VPInstruction::BranchOnCond, {CondVPV},
-              Plan.getVectorLoopRegion()->getCanonicalIV()->getDebugLoc());
+              Plan.getVectorLoopRegion()->getCanonicalIVInfo().getDebugLoc());
```

Comment:
```suggestion
              Plan.getVectorLoopRegion()->getCanonicalIVDebugLoc()
```
avoiding having getCanonicalIVInfo() public?